package.json

```diff
@@ -37,7 +37,7 @@
     "build:dev": "node scripts/babelcheck.js && npm run build:res && npm run build:bundle:dev",
     "dist": "scripts/package.sh",
     "start:res": "node scripts/copy-res.js -w",
-    "start:js": "webpack-dev-server -w --progress",
+    "start:js": "webpack-dev-server --output-filename=bundles/_dev_/[name].js -w --progress",
```

Comment:
Think we'll need this for non-entry chunks, which looks like `--output-chunk-file`?